src/packages/documents/document-blueprints/repository/detail/document-blueprint-detail.server.data-source.ts

```diff
@@ -133,7 +135,7 @@ export class UmbDocumentBlueprintServerDataSource implements UmbDetailDataSource
 			id: model.unique,
```

Comment:
Is this the correct create model? Should it be 
`CreateDocumentBlueprintRequestModel` ?

Reply:
Ohh, didn't notice that before. You may be right, let me see.

Reply:
Found the same problem with the update model (`UpdateDocumentBlueprintRequestModel`), albeit that didn't require a template. But now we use the correct models at least.